### ISSUE TITLE
How to get liquid Nitrogen/Hobo logging

### ISSUE THREAD

**Issue by grace-ac** [sr320/LabDocs#604]
I'm going to Taylor Shellfish Hatchery tomorrow to meet Rhonda @Ellior2 . Was asked to bring liquid nitrogen. Does anyone know how to get it?

Or, @Ellior2, would a cooler with dry ice work?

**Comment by Ellior2** [comment]
Well the reason I like liquid nitrogen is because it will last for a whole month of sampling. I don't think dry ice will last longer than a day or two. I have gone to Airgas in Bremerton in the past with our dewar or you could get it from the UW. @kubu4 or @sr320 could give you tips on how to get it.

Also important- I would like to hook up these hobos you gave me. Do you guys have a logger and all the necessary equipment/manuals to get them going? I have never used them before...

**Comment by seanb80** [comment]
We have everything to set them up. How can be found at

https://seanb80.github.io/blog/2016/12/20/Prepping-HOBO-loggers

**Comment by sr320** [comment]
UW Chem Dept provides LN, but you have to be trained to dispense. Maybe Grace could go there in morning?

You have a -80? Correct?

**Comment by Ellior2** [comment]
I just checked in with the guys at Bremerton Airgas. They have about 80L, plenty for our needs.. They open at 8AM. You could take the Seattle to Bremerton Ferry and stop by Airgas on the way to the hatchery. This is an option, or you can try to figure out the UW situation...

We do have a -80C. I haven't gone down there in a while to look at it but will do so tomorrow.

**Comment by sr320** [comment]
Bremerton Airgas would probably be simpler and faster as stock room might
not even open until 9...

**Comment by Ellior2** [comment]
I also briefly looked at the hobos today and they look much different than the ones @seanb80 writes about. The ones I got measure temperature AND light...  Is the same equipment used to set them up?

**Comment by seanb80** [comment]
I'm not sure, but I can look in to it!

**Comment by sr320** [comment]
We really only have one kind.

**Comment by hputnam** [comment]
http://www.onsetcomp.com/products/communications/base-u-4
The base station should come with all the couplers you need and list the number for the logger type on the side.

If you can't get the couplers but have the base station, you can just align the optics and hold or tape them there to read out the data. You have to align the dot and black square on the front of the logger with the matching set on the base station. http://www.onsetcomp.com/products/data-loggers/ua-002-08

I would highly recommend logging temp with all loggers in the same tank directly next to each other for at least 24h for cross validation prior to allocating them to separate tanks.  Also as Sean says make sure to sync them all to start at the same time and log at the same interval, or your data will not be comparable.

**Comment by grace-ac** [comment]
Okay, so it sounds like I have to stop by FTR to pick up Dewar to use at
Airgas. Anything else from FTR?

**Comment by sr320** [comment]
Logger

**Comment by sr320** [comment]
@Ellior2 
Note your hobos should already be logging. Sean set them up.
You can confirm with blink pattern (see manual sheet)

**Comment by hputnam** [comment]
blinking red light means logging. you can also check detailed status using hoboware pro to confirm logging settings

**Comment by grace-ac** [comment]
Well i just made a huge mistake- misheard the guy at the ferry. Bremerton
ferry doesnt come until 10am!!! I just bought the ticket and everything
:(((((( i should have just drove around.

**Comment by grace-ac** [comment]
That's probably why my navigation didn't give me the ferry option! It takes
longer than driving.

**Comment by seanb80** [comment]
They have in the past let me get a refund if I decided not to take the ferry after queuing up, might be worth a shot!

**Comment by sr320** [comment]
Or you could just relax, grab a coffee, and catch some of lab meeting at https://hangouts.google.com/hangouts/_/6z4e4irui5a4dov6td22cmsacme
:)

**Comment by Ellior2** [comment]
I see no sign of life on these hobos. No blinking lights. Manual says it will blink once every one to four seconds (the shorter the logging interval, the faster the light blinks), but then it says it blinks when it's logging a sample. What if it is set up to log once an hour? Am I missing the blink?

**Comment by seanb80** [comment]
Hm, I'm pretty sure I set them up, but it should be easy to check with the logger base once Grace brings it. I sent her with a serial number for the Hoboware software so you can download the software and use it where convenient.

**Comment by Ellior2** [comment]
So I downloaded the Hoboware for Windows but was not able to get the upgrade to Hoboware pro. When I clicked on the Hoboware Pro link, it brought me to a page where I could purchase it for $100 but I couldn't figure out how to enter in the product key. I registered and got an account and everything, tried putting it in my cart and looking for a place to enter the code but couldn't figure it out. 

I tried to plug the shuttle in (without pendant) and use the program I downloaded: Hoboware for Windows. I don't see the "Sync Shuttle Clock" button that @seanb80 talks about and when I try to "select device" I get this...

![image](https://cloud.githubusercontent.com/assets/20071030/25765375/9d8e6e04-31a1-11e7-8bc7-be24716546fb.png)
